tpe/lib/src/Model.hh

```diff
@@ -47,6 +47,8 @@ class IGNITION_PHYSICS_TPELIB_VISIBLE Model : public Entity
   /// \return Newly created Link
   public: Entity &AddLink();
 
+  public: Entity &GetCanonicalLink();
```

Comment:
can you add doxygen comment for this function?

Reply:
added https://github.com/ignitionrobotics/ign-physics/pull/45/files#diff-a2e564da4da991a87f531e647b213a9cR50-R52